### DISCUSSION TITLE
nginx igress issue

### DISCUSSION THREAD

**Discussion by mxar1** [eclipse-edc/Connector#4618] [Q&A]
We Followed  all the instructions to  deploy the Minimal Viable dataspace.
Unfortunately we noticed that the ingress-nginx-controller pod displays a number of error messages like the following:

W1110 23:17:33.925681      13 controller.go:1109] Error obtaining Endpoints for Service "mvd/provider-manufacturing-vault": no object matching key "mvd/provider-manufacturing-vault" in local store

W1110 23:17:33.929222      13 controller.go:1109] Error obtaining Endpoints for Service "mvd/provider-qna-vault": no object matching key "mvd/provider-qna-vault" in loca

W1110 23:17:37.180466      13 controller.go:1109] Error obtaining Endpoints for Service "mvd/provider-qna-vault": no object matching key "mvd/provider-qna-vault" in local store

W1110 23:17:37.180517      13 controller.go:1109] Error obtaining Endpoints for Service "mvd/provider-manufacturing-vault": no object matching key "mvd/provider-manufact

W1110 23:18:13.488655      13 controller.go:1109] Error obtaining Endpoints for Service "mvd/provider-qna-vault": no object matching key "mvd/provider-qna-vault" in local store

W1110 23:18:13.488714      13 controller.go:1109] Error obtaining Endpoints for Service "mvd/provider-manufacturing-vault": no object matching key "mvd/provider-manufacturing-vault" in local store

W1110 23:18:16.824520      13 controller.go:1109] Error obtaining Endpoints for Service "mvd/provider-qna-vault": no object matching key "mvd/provider-qna-vault" in local store

W1110 23:18:16.825343      13 controller.go:1109] Error obtaining Endpoints for Service "mvd/provider-manufacturing-vault": no object matching key "mvd/provider-manufacturing-vault" in local store

What is the cause of the errors?

Thanks in Advance
Marios

**Comment by paullatzelsperger** [comment]
you provided zero information about your environment, like OS, various versions you used etc. and on top, it does not look like an MVD issue, rather it seems to have to do with your local setup. 

Generally, I have seen instances where one (or all) of recreating the cluster, redeploying the ingress and redeploying MVD may fix things.

**Comment by mxar1** [comment]
Hi, thanks for the quick answer.

Many apologies for the missing information.

We have deployed MVD to  Ubuntu 24.04.1 LTS (GNU/Linux 6.8.0-45-generic x86_64)

We have followed the instructions described in https://github.com/eclipse-edc/MinimumViableDataspace.

We use the Java 17+ version and gradle 8.10 version.

All the pods in the mvd namespace look that they have no problem, no issue found in log files.

Thanks in advance

Marios

**Reply by paullatzelsperger** [reply]
like I said, it must have to do with your KinD installation of config thereof or your host system then. Nothing I can do unfortunately.

**Comment by mxar1** [comment]
Hi, I will check again the configuration.

I hope I will address the issue.

Many thanks 

Best Regards

Marios
